docs/proposals/00_openyurt-glossary.md

```diff
@@ -0,0 +1,64 @@
+# Table of Contents
+
+This document lists terms for the OpenYurt implementation.
+
+[C](#c) | [E](#e) | [N](#n) | [U](#u) | [Y](#y)
+
+# C
+---
+
+### CloudNode
+
+The node that runs on the cloud. The control-plane and other cluster management components are usually running on the CloudNode.
+
+# E
+---
+
+### EdgeNode
+
+The node that is accessible to the edge device. The EdgeNodes are usually located in a sub-optimal network environment. They may be disconnected from the cloud node at any time.
+
+### End User
+
+Represents a user of the OpenYurt cluster.
+
+# N
+---
+
+### NodePool
+
+The CRD represents a pool of edge nodes in the same network region.
+
+# U
+---
+
+### UnitedDeployment
+
+The CRD defines the way of deploying homogeneous workloads with different versions/configurations by NodePools.
+
+# Y
+---
+
+### YurtControllerManager
+
+A replacement of the nodelifecycle controller that prevents the APIServer from marking disconnected EdgeNodes as unready.
```

Comment:
Controller Manager for OpenYurt like Kube Controller Manager for K8s, and include nodelifecycle controller at present, more controllers will be added in the future.